admin/admin.coffee

```diff
@@ -420,6 +441,72 @@ Meteor.methods
     TurkServer.Assignment.getAssignment(asstId).refreshStatus()
     return
 
+  "ts-admin-reject-assignment": (asstId, msg) ->
+    TurkServer.checkAdmin()
+    check(asstId, String)
+
+    TurkServer.Assignment.getAssignment(asstId).reject(msg)
+    return
+
+  # Count number of submitted assignments in a batch
+  "ts-admin-count-submitted": (batchId) ->
+    TurkServer.checkAdmin()
+    check(batchId, String)
+
+    result = {}
+    # First refresh everything
+    Meteor.call "ts-admin-refresh-assignments", batchId, (err, res) ->
```

Comment:
Interestingly, this is going to achieve the opposite of the effect you desired, because it will call the method asynchronously and therefore the count will not be stored after the method returns.

See https://docs.meteor.com/#/full/meteor_call